orientdb/src/test/resources/hibernate.properties

```diff
@@ -1,18 +1,14 @@
-# Copyright (C) 2016 Hibernate.
 #
-# This library is free software; you can redistribute it and/or
-# modify it under the terms of the GNU Lesser General Public
-# License as published by the Free Software Foundation; either
-# version 2.1 of the License, or (at your option) any later version.
+# Hibernate OGM, Domain model persistence for NoSQL datastores
 #
-# This library is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-# Lesser General Public License for more details.
+# License: GNU Lesser General Public License (LGPL), version 2.1 or later
+# See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
 #
-# You should have received a copy of the GNU Lesser General Public
-# License along with this library; if not, write to the Free Software
-# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
-# MA 02110-1301  USA
 
 hibernate.ogm.datastore.provider = ORIENTDB
+hibernate.connection.url = jdbc:orient:memory:test
+javax.persistence.jdbc.url = jdbc:orient:memory:test
+javax.persistence.jdbc.driver=com.orientechnologies.orient.jdbc.OrientJdbcDriver
+javax.persistence.jdbc.user=admin
+javax.persistence.jdbc.password=admin
```

Comment:
Because at the moment we only work with the embedded DB the user shouldn't have to set this properties.
The only optional property  should be the name of the db to use.